Description: Remove backtrace dep; it's not in Debian
Author: John Goerzen <jgoerzen@complete.org>
Last-Update: 2022-10-22
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -38,7 +38,6 @@
 
 [dependencies.anyhow]
 version = "1.0.57"
-features = ["backtrace"]
 
 [dependencies.bytes]
 version = "1.1.0"
